### DISCUSSION TITLE
How to disable "telescope integration not enabled" error message.

### DISCUSSION THREAD

**Discussion by liujoey** [NeogitOrg/neogit#656] [General]
I'm using `fzf-lua` as my main fuzzy finder, I don't have telescope because it is too slow for my large project. And I use `stevearc/dressing.nvim` to set `fzf-lua` as a replacement of `vim.ui.select`, but neogit just doesn't like that idea, it constantly popup error messages telling me that telescope integration was disabled. Is there a configuration that I can disable this error message? Thank you.

**Comment by ten3roberts** [comment]
This is an error on our part.

The "check if enabled" function has that logging side effect 

https://github.com/NeogitOrg/neogit/blob/94b5d98a3381594e91c445842192f1d2f1375b96/lua/neogit/lib/finder.lua#L125

**Reply by liujoey** [reply]
Do you plan to provide a config to silent this message? Thank you for the great software by the way.

**Reply by ten3roberts** [reply]
Yes, I do https://github.com/NeogitOrg/neogit/pull/658 😊 

I am not using the telescope integration either, as I find the more minimal `dressing` approach using telescope without the preview buffer and all but just a simple dropdown sufficient.
